### ISSUE TITLE
Add .prettierrc and .prettierignore by default when do `ng new app-name`

### ISSUE THREAD

**Issue by kasir-barati** [angular/angular-cli#23398]
### Description

Hello Angular dev team. This is just my opinion and it may seems very opinionated and I cannot proclaim that it is not. BTW I prefer `ng new app-name` generated and install prettier as devDep for me and create a compatible `.prettierrc`, `.prettierignore` file  for me.

## What motivates me?

When I create a new project I always had to do these tedious steps:
1. `pnpm add -D prettier`
2. `code .prettierrc` and copy paste configurations
3. `code .prettierignore` and copy paste configurations
4. add this script in `package.json`: `"format": "prettier --write '**.json' '**.ts' '**.js'"`

### Proposed solution

It is simple - TBH I mean I have no proposal for it - IMO, In the steps that generates boilerplates add prettier too.

### Alternatives considered

I can write a custom simple bash script that does that for me. e.x.

```cmd
my-ng new app-name
```

**Comment by kasir-barati** [comment]
Thanks @JoostK for transferring my issue, I am sorry for that. BTW here is `my-ng` implementation. As you can see it is a little time consuming and if we had it in place I guess it would be more vivider this script

https://github.com/kasir-barati/the-pragmatic-programmer/blob/main/customize-your-dev-env/my-ng.md

**Comment by alan-agius4** [comment]
Duplicate of https://github.com/angular/angular-cli/issues/7379

TLDR; there is already a 3P schematic https://github.com/schuchard/prettier-schematic. You can also create your own schematic by extending the default ng-new schematic.

**Comment by kasir-barati** [comment]
@alan-agius4 I do not think this is a good solution, I mean as if you read my script I did and I meant something else, Being `format` script in the `package.json` and prettier as devDep is far simpler than configuring lint-staged/husky and it is a must for `@angluar/cli` IMO.

I am not convinced.
